src/soc/ibm/power9/pstates.c

```diff
@@ -324,6 +440,154 @@ static void update_resclk(int ref_freq_khz)
 	}
 }
 
+static int32_t wof_find(struct wof_image_entry *entries, uint8_t entry_count,
+			uint32_t core_count,
+			const struct voltage_bucket_data *poundV_bucket)
+{
+	const struct region_device *wof_device = wof_device_ro();
+
+	const uint16_t socket_power_w = be16toh(poundV_bucket->sort_power_normal);
+	const uint16_t sort_power_freq_mhz = be16toh(poundV_bucket->nominal.freq);
```

Comment:
After compared the results with dumped HOMER I discovered that I made and error while analyzing this code. Turbo mode should be used instead of nominal, the attribute that controls it have a default value specified in two different places and I have chosen the wrong one.